ROMFS/px4fmu_common/init.d-posix/airframes/CMakeLists.txt

```diff
@@ -91,6 +91,7 @@ px4_add_romfs_files(
 	4016_gz_x500_lidar_down
 	4017_gz_x500_lidar_front
 	4018_gz_quadtailsitter
+	4030_gz_tiltrotor
```

Comment:
i think generally we just continue the numbering, so it would be
`4019_gz_tiltrotor`

Reply:
updated